### ISSUE TITLE
Contradictions from no-potential-path-strings and no-unnecessary-curly-strings

### ISSUE THREAD

**Issue by ijlee2** [ember-template-lint/ember-template-lint#3057]
I'd like to pass a string that starts with `@` to an argument:

```hbs
<Ui::Demo @title="@my-org/my-package">
  ...
</Ui::Demo>
```

In `ember-template-lint@v5`, the code above caused `no-potential-path-strings` to fail. So I used double curly braces to avoid false positives.

```hbs
<Ui::Demo @title={{"@my-org/my-package"}}>
  ...
</Ui::Demo>
```

In `ember-template-lint@v6.0.0`, using double curlies for string causes `no-unnecessary-curly-strings` to fail. And without double curlies, `no-potential-path-strings` still marks the code as an issue.

```sh
error  Potential path in attribute string detected. Did you mean {{@my-org/my-package}}?  no-potential-path-strings
```

Could either/both rule(s) be reviewed for accuracy and/or relevance?

**Comment by ijlee2** [comment]
Looking at #1457, I feel like `no-potential-path-strings` had been added to account for rare cases.

**Comment by lifeart** [comment]
@ijlee2 thank you for your report!
